releasenotes/notes/41858.yaml

```
@@ -0,0 +1,6 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: installation
+releaseNotes:
+  - |
+    **Fixed** only print warning if istio-cni is not the default cni plugin when cni is used as a standalone plugin
```

I leave the logic to the networking team, but is this better wording for the release note:
```suggestion
    **Removed** warning if istio-cni is not the default CNI plugin when CNI is used as a standalone plugin.
```

---

@ericvn release note is updated

---

@ericvn could you mind reviewing this pr again?